pkg/auth/artifact_verifier_test.go

```diff
@@ -37,6 +37,26 @@ func buildTestFileTree(t *testing.T, files []testFile) string {
 	return tempDir
 }
 
+// This is copied from pkg/artifact/location_data.go to avoid an import cycle
+func VerificationDataForLocation(location *url.URL) VerificationData {
```

Comment:
I anticipate this function being defined on an `ArtifactRegistry` struct in the near future